README.md

```diff
@@ -32,8 +32,27 @@ You'll also need to clone a new `.env` file from the `.env.template` to store lo
 $ cp .env.template .env  # (first time only)
 ```
 
+You will need to create a Trello account and generate and API Key and Token. Follow the instructions in the below links:
+    [https://trello.com/signup]
+    [https://trello.com/app-key]
+
+Once you have an account, create some cards (at least one in To Do list.)
+
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+Set the following value in the `'.env` file once you have got these from Trello.
+    [API_KEY=]
+    [API_TOKEN=]
```

Comment:
Minor, but it would be nice to add these to your `.env.template` (not with the actual secret values!)